src/views/register/register.tsx

```diff
@@ -0,0 +1,160 @@
+                            <p className="text-white ml-3">
+                                <span style={{ color: '#90A4AE' }}>Real-time Monitoring. </span>Track <br /> Ethereum network hash-rate and
+                                <br /> difficulty in real-time with charts and <br /> historical data.
+                            </p>
+                        </div>
+                        <div style={{ display: 'flex', alignItems: 'center' }} className="mt-3">
+                            <Image src={check} alt="logo" className="text-center mt-2" style={{ marginTop: '-66px' }} />
+                            <p className="text-white ml-3">
+                                <span style={{ color: '#90A4AE' }}>Miner Distribution Analysis.</span>
+                                <br />
+                                Analyze miner distribution on the
+                                <br />
+                                network to understand risks to <br />
+                                security and decentralization.
+                            </p>
+                        </div>
+                        <div style={{ display: 'flex', alignItems: 'center' }} className="mt-3">
+                            <Image src={check} alt="logo" className="text-center mt-2" style={{ marginTop: '-66px' }} />
+                            <p className="text-white ml-3">
+                                <span style={{ color: '#90A4AE' }}>Miner Distribution Analysis.</span>
+                                <br />
+                                Analyze miner distribution on the
+                                <br />
+                                network to understand risks to <br />
+                                security and decentralization.
+                            </p>
+                        </div>
+                    </div>
+                    <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+                        <div className="sec-box bg-white rounded px-10 mt-8 py-5" style={{ height: '645px', width: '464px' }}>
+                            <p className="text-black text-xl font-weight-bold mt-4 text-center" style={{ fontSize: '1.5rem' }}>
+                                Register
+                            </p>
+                            <button
+                                className="rounded border py-2 mt-6 w-full gap-3"
+                                style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', justifyContent: 'center' }}
+                            >
+                                <Image src={google} alt="logo" />
+                                <p className="text-black font-weight-bold">LOGIN WITH GOOGLE</p>
+                            </button>
+                            <button
+                                className="rounded border py-2 text-black font-weight-bold mt-2 w-full gap-3"
```

Comment:
Login with google and login with github look different. Their shadows look different. Please fix that.